content/en/news/2019/announcing-1.3.5/index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Announcing Istio 1.3.5
+description: Istio 1.3.5 release announcement.
+publishdate: 2019-11-11
+attribution: The Istio Team
+subtitle: Minor Update
+release: 1.3.5
+aliases:
+    - /news/announcing-1.3.5
+---
+
+This release includes bug fixes to improve robustness. This release note describes what's different between Istio 1.3.4 and Istio 1.3.5.
+
+{{< relnote >}}
+
+## Bug fixes
+
```

Comment:
We should put this one on top:
```
- **Fixed** an issue where Envoy may consume excessive CPU due to incorrectly handling listener filter timeouts. ([Issue 18229](https://github.com/istio/istio/issues/18229))
```
It's probably the most important fix in this release.

Reply:
@rlenglet please see the latest update and let me know your thoughts. 

Also @geeknoid This varies a bit in the normal announcement style since this has both bug and security fixes. Please let me know if you have any suggestions.